app_dart/lib/src/request_handlers/flush_cache.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
2020.

Reply:
This work is copied from work I did in 2019, so should be based on 2019.

Reply:
I am not sure if the copyright is based on the file created time, or the earlier work it derives from. @godofredoc to confirm.

Reply:
See https://github.com/flutter/cocoon/pull/701#discussion_r395447338 for context from hixie